R/diagnostics.R

```diff
@@ -19,6 +19,10 @@
 #'   tolerance set by {tol}, FALSE otherwise.
 #' @export
 isTimestepRegular <- function(dates, hist=TRUE, tol=.Machine$double.eps^0.5, handler=stop) {
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  aes=TimeInterval=geom_histogram=xlab=NULL
```

Comment:
In general for this visible binding error, I've started using ".ggplot2.var", ".dplyr.var", etc. rather than NULL to make it a tiny bit clearer why it needs to be done. That change might be a good idea for TimeInterval - @miguelcleon, would you mind changing it, please?

I'm also pretty sure there's a better fix for aes, geom_histogram, and xlab; could you please try changing the roxygen line to `@importFrom ggplot2 ggplot aes geom_histogram xlab` and removing them from line 24? Those shouldn't ever need to be treated as global variables.